services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/monitor/core.py

```diff
@@ -0,0 +1,394 @@
+# here services can be registered to be monitored for their
+# current status. Events are triggered base on what happens
+# Monitoring for:
+# - servie available
+#
+#
+# Handlers to subscribe to this thing
+
+import asyncio
+import logging
+from asyncio import Lock, sleep
+from typing import Deque, Dict, Any, Optional
+
+from aiohttp.web import Application
```

Comment:
You are mixing now ``FastApi`` and ``aiotthp``. I guess here it is 
```python
from fastapi import FastAPI
# from aiohttp.web import Application WRONG!
```

Reply:
yes, thanks for this. Somehow I've got them mixed up a while ago and forgot about them.